aws-go-assume-role/assume-role/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"github.com/pulumi/pulumi-aws/sdk/v2/go/aws"
+	"github.com/pulumi/pulumi-aws/sdk/v2/go/aws/s3"
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi/config"
+)
+
+func GetRegion(ctx *pulumi.Context) string {
+	v, err := config.Try(ctx, "aws:region")
+	if err == nil {
+		return v
+	} else {
+		return err.Error()
+	}
+}
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+
+		config := config.New(ctx, "")
+		roleToAssumeARN := config.Require("roleToAssumeARN")
+
+		region := GetRegion(ctx)
```

Comment:
I haven't tried this, but I think it would allow your `GetRegion` function to be deleted:


```suggestion
		region := config.Require(ctx, "aws:region")

		c := config.New(ctx, "")
		roleToAssumeARN := c.Require("roleToAssumeARN")
```

Reply:
Changed it with new pull request with new conversions